Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -863,23 +817,23 @@ void Wiimote::Update()
   }
 }
 
-void Wiimote::ControlChannel(const u16 _channelID, const void* _pData, u32 _Size)
+void Wiimote::ControlChannel(const u16 channelID, const void* pData, u32 Size)
```

Comment:
Since you're going to fix the naming, why not change these to channel_id, data, size?